web/src/js/__tests__/ducks/tutils.js

```diff
@@ -7,3 +7,30 @@ export function createStore(parts) {
         applyMiddleware(...[thunk])
     )
 }
+
+export function TFlow(intercepted=false, marked=false, modified=false) {
+    return {
+        intercepted : intercepted,
+        marked : marked,
+        modified: modified,
+        id: "foo",
+        request: {
```

Comment:
Only nontrivial comment on this PR: We should have a complete flow - with all attributes - here right away. Ideally autogenerated from the Python side so that it's easy to update.

A bit unorthodox, but we could write a Python test that creates/updates `__tests__/ducks/_tflow.js` with the contents of `mitmproxy.tools.web.app.flow_to_json(tflow())` (+ `export default ...` around it)? We can then just import that here in tutils and re-export it.

(let's do that in a separate PR, but do it now before much other stuff depends on TFlow)